### PR TITLE
Fix specs broken by aruba changes

### PR DESCRIPTION
**Pull request by dgynn**
This PR fixes the specs broken by the deprecation of `check_file_presence` in the latest aruba. While I was in there I also cleaned up the other deprecation warnings from aruba, rspec, and active record.

### CODE REVIEW

**Comment by dgynn** [comment]
I've also updated the travis config to allow travis's bundler caching to take effect and to run the allow_failures rubies after the primary rubies. It looks like this will allow travis to run about 50% faster.

File: Appraisals [hunk 1]
```diff
@@ -1,14 +1,11 @@
 appraise "3.2" do
-  gem "rails", "~> 3.2.0"
-  gem "paperclip", :path => "../"
+  gem "rails", "~> 3.2.15"
```
Comment (tute) [inline Appraisals]: I'd leave this as `~> 3.2.0`, as bundler will anyway install latest.

**Comment by tute** [comment]
Thank you very much for your work. Can you please address my only comment, squash the commits together into one with a good commit message, and rebase on top of latest master? I'll merge right after. Thank you again!

**Comment by dgynn** [comment]
Thanks, @tute . Squashed, rebased, and back to green.

**Comment by tute** [comment]
That's great, thank you! :clap: :+1:

**Pull request merged**